### PR TITLE
Add caption vvaths

### LINKED ISSUE #2: Activity 1: Your First Caption
Using the API, I have just created a file with a random octocat for you to caption. In this activity, we will learn how to make changes to files in our GitHub repository.

- [ ] @SandyElP
- [ ] @VictorNunezla
- [ ] @chambersDon
- [ ] @darasjohn15
- [ ] @divdee
- [ ] @dsmith1000
- [ ] @janetrobinson123
- [ ] @lonturner
- [ ] @myusernameherenow
- [ ] @niseyLee
- [ ] @sbhairraju
- [ ] @sheikhahmedhorizon
- [ ] @sjmhome07
- [ ] @vvaths
- [ ] @vyas000
- [ ] @BryantHennessy
- [ ] @bpatterson617

### PR DESCRIPTION
**Pull request by vvaths**
fixes #2

### CODE REVIEW

File: _slides/02-vvaths.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Amazing New Caption!!
```
Comment (vyas000) [inline _slides/02-vvaths.md]:
```suggestion
Magic Amazing New Caption!!!!
```

File: _slides/02-vvaths.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
...
```
Comment (vyas000) [inline _slides/02-vvaths.md:7]: good job!

**Review by vyas000 — CHANGES_REQUESTED** [review]
please make changes requested!

File: _slides/02-vvaths.md (hunk 2 above)
Comment (vvaths) [reply _slides/02-vvaths.md:7]: thank you!

File: _slides/02-vvaths.md (hunk 1 above)
Comment (vvaths) [reply _slides/02-vvaths.md]: done @vyas000

**Review by vyas000 — APPROVED** [review]
ELGTM!

**Pull request merged**